stripe/_api_resource.py

```diff
@@ -190,3 +189,6 @@ def _static_request_stream(
 
         response, _ = requestor.request_stream(method_, url_, params, headers)
         return response
+
+
+from stripe import _api_requestor, _error, util  # noqa
```

Comment:
What's the purpose of importing at the bottom of the file like this?

Reply:
One of my attempts at resolving circular references, not needed anymore.

Reply:
Fixed